### PR TITLE
fix(*): style update or fix of several components

### PR DESCRIPTION
**Pull request by sun-tea**
- [ ] Feature
- [x] Fix
- [x] Enhancement

## Brief
Minor visual fixes and improvement in several components.

## Description
Fixed styling and typos. The `placeHolder`  > `placeholder` is a change that must be fixed in captain and sonar, I already took care of that and can open the PRs as soon as the new version of Stardust is published.

## Related / Associated Jira Cards :
Fixes related to feedback from Arthur, and otherwise casual maintainance.

## Todo - Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: src/Dropdown/index.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ class Dropdown extends PureComponent {
     );
 
     return (
-      <div className={className} data-testid="dropdown">
+      <Wrapper className={className} data-testid="dropdown">
```
Comment (spartDev) [inline src/Dropdown/index.js]: You can remove the Wrapper here and put your css directly in `default`export

File: src/Select/index.js [hunk 2]
```diff
@@ -79,22 +81,39 @@ class Select extends PureComponent {
...
+        this.setState({ value: initialValue });
+      } else {
+        this.initializeValue();
+      }
+    } else if (!placeholder) {
+      this.initializeValue();
     }
   }
 
+  initializeValue = () => {
```
Comment (spartDev) [inline src/Select/index.js]: Can you put a little description here

**Comment by sun-tea** [comment]
The current NumberInput does not strip out the non-digits characters when they are entered by the user. I think it makes sense to strip them out automatically, what do you think?

**Comment by spartDev** [comment]
yes I know that bug @leopoldhoudin opened an issue about that #70

**Pull request merged**